src/vl-tabs.js

```diff
@@ -97,7 +97,7 @@ export class VlTabs extends vlElement(HTMLElement) {
   }
 
   __getTabTemplate({id, title}) {
-    const pathname = window.location.pathname;
+    const pathname = this.getAttribute('data-vl-href') || window.location.pathname + window.location.search;
```

Comment:
Duplicatie van `this.getAttribute('data-vl-href') || window.location.pathname + window.location.search`. Dit kan opgevangen worden door een href getter te introduceren.